docs/coding-guidelines/adding-api-guidelines.md

```diff
@@ -43,7 +43,7 @@ the implementation without compat concerns in future releases.
 ## Making the changes in repo
 
 **If changing the library version**
-  - Update the `AssemblyVersion` property in `<Library>\Directory.Build.props` (ex: [System.Runtime\Directory.Build.props](https://github.com/dotnet/corefx/blob/master/src/System.Runtime/Directory.Build.props#L4)) to the version determined above.
+  - Update the `AssemblyVersion` property in `<Library>\Directory.Build.props` (ex: [Microsoft.CSharp\Directory.Build.props](https://github.com/dotnet/runtime/blob/master/src/libraries/Microsoft.CSharp/Directory.Build.props#L4)) to the version determined above.
```

Comment:
I didn't find AssemblyVersion in most of Directory.Build.props files, including the old mentioned one. So, I replaced it with a one that has an AssemblyVersion set in it. Is AssemblyVersion set globally somewhere ? Does that worth mentioning ?

Reply:
@ViktorHofer, can you assist?

Reply:
@joperezr

Reply:
The AssemblyVersion is no longer stated on every project, and instead is calculated by the Arcade SDK on the fly by using the following target: https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.targets#L14-L43

Basically what it does is to either set the version to 42.42.42.42 when building locally, or to calculate the actual assembly version based on 

https://github.com/dotnet/runtime/blob/653cf01273242935f85290a131ccf7dc966e9d37/eng/Versions.props#L5-L8

and on the build version and patch version. You can find more info on how the version is calculated here: https://github.com/dotnet/arcade/blob/master/Documentation/CorePackages/Versioning.md

> Does that worth mentioning ?

I think it would be worth it to update the docs to say that AssemblyVersion isn't always on the Directory.Build.props file any longer (unless we want to override the default for some reason) so that if we need to change assembly version then Directory.Build.props would be the place to do it, and that if anybody wants more info on how is the original version calculated, they can look at the arcade documentation to find more info.

Reply:
@joperezr. Thanks for that information. Can you look at the latest commit?

Reply:
Ok, Joe was faster :)